javascripts/googleAnalyticsImporter.js

```diff
@@ -23,8 +23,10 @@
     (async () => {
       const response = await fetch('/index.php?' + new URLSearchParams(searchParams));
       const data = await response.json();
-      if (data.displayPending){
-        displayPendingNotification(data.availableDate);
+      if ($('.site-without-data').length && data.isGASite) {
+        displayPendingNotification('', 'successMessage');
+      } else if (data.displayPending, '') {
```

Comment:
Why have the `, ''` as part of the `if else`?

Reply:
@snake14 Thanks was a typo mistake